complaints/ccdb/index_ccdb.py

```diff
@@ -50,26 +52,28 @@ def enhance_complaint(complaint):
     if ':updated_at' in complaint:
         return complaint
 
-    # Simulate the Socrata field
-    dt = parse_date(complaint.get("date_received"))
-    complaint[":updated_at"] = time.mktime(dt.timetuple()) if dt else 0.0
+    # "Feature Flag" - Merge in the new metadata
+    if FEATURE_MERGE_NEW_META:
+        # Simulate the Socrata field
+        dt = parse_date(complaint.get("date_received"))
+        complaint[":updated_at"] = time.mktime(dt.timetuple()) if dt else 0.0
```

Comment:
I like the feature flag approach here. We'll need to update the way this is simulated later to use a field other than date_received for this value, but we're squared away on what that will look like. 👍 Thanks!